content/en/docs/tasks/administer-cluster/verify-signed-images.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Verify Signed Container Images
+content_type: task 
+min-kubernetes-server-version: v1.24
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.24" >}}
+
+## {{% heading "prerequisites" %}}
+
+These instructions are for Kubernetes {{< skew currentVersion >}}. If you want to check the
+integrity of components for a different version of Kubernetes, check the documentation for
+that Kubernetes release.
+
+You will need to have the following tools installed:
+- `cosign` ([install guide](https://docs.sigstore.dev/cosign/installation/))
+- Go compiler ([install guide](https://go.dev/doc/install)
+- `curl` (often provided by your operating system)
+
+## Verifying image signatures
+For a complete list of images that are signed please refer to [Releases](/releases/download/).
+
+Let's pick one image from this list and verify its signature
+using `cosign verify` command:
+
+```shell
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+
+{{% alert title="Note" %}}
```

Comment:
I'd  suggest we use the `note` shortcode for consistency with other pages, i.e,

```
{{< note >}}
something
{{< /note >}}
```

Reply:
@tengqm can you share an example where this shortcode is used? My understanding (which could be wrong) is that the alert shortcode was the convention

Reply:
For example, check https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/concepts/containers/images.md

where you can find:

```
{{< note >}}
This is the recommended approach to run containers based on images
in private registries.
{{< /note >}}
```

Reply:
Also please check the website style guide on this: https://kubernetes.io/docs/contribute/style/style-guide/#shortcodes

Reply:
Thank you @tengqm for teaching me something new today!! Updating now.